.azure-pipelines/security-tools.yml

```diff
@@ -53,28 +53,21 @@ jobs:
       outputFormat: sarif
       scanFolder: SecurityTmp
       suppressionsFile: tools/SecurityTools/CredScanSuppressions.json
-
-  - task: PowerShell@2
-    displayName: Copy PDB for BinSkim
-    inputs:
-      targetType: inline
-      script: ./src/lib/pdb/CopyPdbToArtifacts.ps1
-      pwsh: true
   
   - task: PowerShell@2
     displayName: Generate a response text file for BinSkim
     inputs:
       targetType: 'inline'
       script: |
-        New-Item $(Build.SourcesDirectory)/artifacts/MyFileList.rsp -ItemType File -Force
-        (Get-ChildItem -Path .\artifacts\Debug -Include *.dll,*.exe -Exclude msalruntime.dll,msalruntime_arm64.dll,msalruntime_x86.dll -Recurse).FullName > $(Build.SourcesDirectory)/artifacts/MyFileList.rsp
+        New-Item $(Build.SourcesDirectory)/artifacts/ScannedFileList.rsp -ItemType File -Force
```

Comment:
ScannedFileList gives you an impression that files on this list have been scanned, but that's the opposite. Use FilesToScan